frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -198,6 +203,11 @@ export const KEY_MAPPING: KeyMappingInterface = {
             description: 'User interactions that were automatically captured.',
             examples: ['clicked button'],
         },
+        $autocapture_disabled_server_side: {
```

Comment:
BTW why do we need to send this _back_ to PostHog? I definitely see how it makes sense to save this in posthog-js's persistence layer, but not quite why we send this along with the events too @raquelmsmith

Reply:
I don't think we need it, I think I was just copying the implementation and assumed we sent everything back.